src/handle-initial-jsmd.js

```diff
@@ -2,9 +2,12 @@ import { stateFromJsmd } from './tools/jsmd-tools'
 import handleUrlQuery from './tools/handle-url-query'
 import { updateAppMessages, importInitialJsmd, evaluateAllCells } from './actions/actions'
 
+const SERVER_QUERY_STRINGS = ['revision']
+
 export default async function handleInitialJsmd(store) {
   let state
-  if (window.location.search) {
+  if (window.location.search &&
+    SERVER_QUERY_STRINGS.every(s => !window.location.search.includes(s))) {
```

Comment:
This feels kind of brittle to me. I wonder if we'd do better to only handle the url query for the case that we're in standalone (non-server) mode.

Reply:
Very good point. I think I'll just check `IODIDE_BUILD_TYPE` as we do in other source files.